src/foundation.cs

```diff
@@ -875,8 +875,76 @@ public enum NSAttributedStringNameKey {
 
 		[Field ("NSReplacementIndexAttributeName")]
 		ReplacementIndex,
+
+		[Watch (10, 0), TV (17, 0), Mac (14, 0), iOS (17, 0), MacCatalyst (17, 0)]
+		[Field ("NSInflectionAgreementArgumentAttributeName")]
+		InflectionAgreementArgument,
+
+		[Watch (10, 0), TV (17, 0), Mac (14, 0), iOS (17, 0), MacCatalyst (17, 0)]
+		[Field ("NSInflectionAgreementConceptAttributeName")]
+		InflectionConcept,
```

Comment:
This matches the naming for the previous value

```suggestion
		InflectionAgreementConcept,
```